### DISCUSSION TITLE
Include user icon next to a media file

### DISCUSSION THREAD

**Discussion by cdorsat** [mediacms-io/mediacms#814] [Feature Requests]
The ICON corresponding to the user is missing in front of the user name on the home page. Can it be increased?

**Comment by mgogoulos** [comment]
Hi, can you share a screenshot, or a link on demo.mediacms.io ? I can't see what you describe.  
Thanks

**Comment by cdorsat** [comment]
You can refer to the red circle marked in the picture below. I think it makes people look more comfortable.

![Youtube compare](https://user-images.githubusercontent.com/22068633/232226738-93fe5e22-1a44-4d1f-aacc-050afcb1d2dd.png)

![User Icon](https://user-images.githubusercontent.com/22068633/232226775-400e4d95-1b7e-4b05-a7ac-dfe06577714e.png)
